cpu/stm32/kconfigs/Kconfig.clk

```
@@ -28,31 +28,53 @@ config USE_CLOCK_HSI
 
 endchoice
 
-if CPU_FAM_G0 || CPU_FAM_G4
+if CPU_FAM_L4 || CPU_FAM_L5 || CPU_FAM_WB
+choice
+bool "Source clock for PLL" if USE_CLOCK_PLL
+default CLOCK_PLL_SRC_MSI if !BOARD_HAS_HSE
+default CLOCK_PLL_SRC_HSE if BOARD_HAS_HSE
```

I think this can be simplified to
```suggestion
default CLOCK_PLL_SRC_HSE
```